configs/fakedb/ingestCfgs/fakedb_ForcedSource.json

```diff
@@ -0,0 +1,17 @@
+{"database":"fakedb",
+    "table":"ForcedSource",
+    "is_partitioned":1,
+    "chunk_id_key":"chunkId",
+    "sub_chunk_id_key":"subChunkId",
+    "is_director":0,
+    "director_key":"objectId",
+    "schema":[
+     {"name":"objectId","type":"BIGINT NOT NULL"},
+     {"name":"ccdVisitId","type":"BIGINT NOT NULL"},
+     {"name":"psFlux","type":"DOUBLE"},
+     {"name":"psFlux_Sigma","type":"DOUBLE"},
+     {"name":"chunkId","type":"INT UNSIGNED NOT NULL"},
+     {"name":"subChunkId","type":"INT UNSIGNED NOT NULL"}
+    ],
```

Comment:
I'm quite confused about the definition of the "schema" above.  See https://json-schema.org/learn/miscellaneous-examples.html of what I used to see how schemas defined in JSON.

Reply:
It would be nice if it was db_schema or something to separate it from json schema, but these are configuration files for the ingest system, so I can't really change them.